src/com/gmail/chernii/oleksii/future/FutureExample.java

```diff
@@ -0,0 +1,25 @@
+package com.gmail.chernii.oleksii.future;
+
+import java.util.concurrent.*;
+
+/**
+ * Created by Space on 19.03.2019.
+ */
+public class FutureExample {
+    private static String YES ="YES";
```

Comment:
It is not constant! You forgot modifier final.
![image](https://user-images.githubusercontent.com/20267148/54723436-376fb980-4b70-11e9-9e47-7d148b13dd9e.png)